service/grpc/option.go

```diff
@@ -31,6 +37,14 @@ func newOptions(opts ...Option) Options {
 		o(&opt)
 	}
 
+	if opt.Client == nil {
+		opt.Client = grpc.NewClient()
+	}
+
+	if opt.Transport == nil {
+		opt.Transport = grpcT.NewTransport()
```

Comment:
@butonic our package now defaults to grpc transport and grpc client on the creation of the service. This can be changed when creating a new services, for instance:

```go
service := grpc.NewService(
	grpc.Name(options.Name),
	grpc.Context(options.Context),
	grpc.Address(options.Address),
	grpc.Namespace(options.Namespace),
	grpc.Logger(options.Logger),
	grpc.Transport(grpctransport.NewTransport()),
	grpc.Client(mgrpc.NewClient()),
)
```